### ISSUE TITLE
UI alignment across our key websites

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/foundation.mozilla.org#582]
Properties should be better linked with smoother UX. UI should be aligned, but can differ.

**Comment by xmatthewx** [comment]
One opportunity is to put a link on Site homepage getinvolved section that takes a visitor the pulse help categories. Might require a new help landing page in pulse, similar to our issues landing page.

**Comment by kristinashu** [comment]
@xmatthewx based on our meeting, I think we should separate this epic into two: `Align UI` + `Link Sites`

**Comment by xmatthewx** [comment]
Sounds good to me. 

I'm also questioning the decision to do links after UI. Seems useful (and not too complicated) to build connections pre-MozFest, even if UI still varies. What do you think?

**Comment by kristinashu** [comment]
For a more seamless and less jarring experience, I think sites should feel more aligned before we make people jump back-and-forth between them. Although technically easy to link the sites, UX-wise it might to tough to pull it off smoothly. But having says that, adding links to the footer could be a small easy one to do.

**Comment by kristinashu** [comment]
Aaaaand https://www.mozilla.org/en-US/ has changed again...

**Comment by xmatthewx** [comment]
Wow. At least it's slightly more in line with the internet health page and things on our side. The top nav is growing, has an odd mix of hierarchy. 

Dang. Can we replace that Teach the Web link to Learning with a link to the new site?

**Comment by kristinashu** [comment]
(copied from https://github.com/mozilla/network/issues/751)

Based on the kickoff meeting, we will first focus on UI alignment and cross-linking will come later.

Next steps include: 
- [ ] update [doc](https://docs.google.com/a/mozilla.com/presentation/d/1R1r4RCfoxKYAPVZerAPItJk1JZuALrc5jWm0t1QMijU/edit?usp=sharing) with moz.org designs
- [ ] check with moz.org designers on where their design is heading
- [ ] prioritize order of UI updates listed in the doc
- [ ] open tickets for each UI element to make the design decisions
- [ ] work with devs to come up with a plan of attack for implementation
- [ ] check with PMs on timeline and resourcing

**Comment by kristinashu** [comment]
(copied from #770)

These are tickets for the first set of design work that needs to be done. Will wait until these are finished before moving on to opening the other tickets.

#### Header Section (Wordmark, Nav, Secondary nav)
- [[MoFo Unified Design] Header and navigation](https://github.com/MozillaFoundation/Design/issues/241)

#### Thumbnails & Feature boxes
- [[MoFo Unified Design] Feature and thumbnail](https://github.com/MozillaFoundation/Design/issues/242)

#### Dividers
- [[MoFo Unified Design] Dividers and section titles](https://github.com/MozillaFoundation/Design/issues/243)

#### Buttons & links
- [[MoFo Unified Design] Buttons and links](https://github.com/MozillaFoundation/Design/issues/244)

#### Hold off on
- Fonts (Switch to Zilla and Zilla Highlight #303)
- Footer
- Newsletter signup
- Forms & Notifications/alerts
- Icon style
- Color
- Banner

**Comment by kristinashu** [comment]
Added some initial comps to redpen https://redpen.io/p/rx51f171fd150ae1fc

**Comment by kristinashu** [comment]
@xmatthewx let me know if you have any feedback on the comps https://redpen.io/p/rx51f171fd150ae1fc

**Comment by xmatthewx** [comment]
Looks fantastic! Really quick, strong work @kristinashu. I really appreciate it. We can have @jessevondoom take a look at [redpen](https://redpen.io/p/rx51f171fd150ae1fc) and then make a plan with Alan for implementation.

**Comment by jessevondoom** [comment]
These look amazing. I only had the most minor bits of feedback in redpen. Think we're ready to talk with @alanmoo about implementation.

**Comment by kristinashu** [comment]
I meet with @alanmoo, @mmmavis and @gvn. Our next steps are: 

#### Open Pulse tickets for Mavis:
- **Nav updates**: gradient divider instead of solid pink, ‘add post’ button similar to Network’s ‘sign up’ CTA, font weight (Blocked by design - TBD: mobile nav for Pulse, sticky like Network?, [new new new moco logo guidelines](https://mozilla.ninja/expression/))
- **Background:** white instead of grey https://github.com/mozilla/network-pulse/issues/700
- **Font updates:** thumbnail titles black instead of pink, no intro text, body text links black with underline instead of blue (need ticket)

#### Open Network tickets for Gavin:
- **Nav updates**: show nav items instead of hamburger, update responsive states (need ticket)
- **Font updates:** remove highlight from banner titles and update all Arvo to Zilla. @taisdesouzalessa is probably best to oversee the Zilla update since she is most familiar with the styles (need ticket)
- **Dividers:** update dashed divider to gradient + highlight text
- **Home page features:** update Get Involved layout to look more like News layout (this is closely related to the re-balancing work https://github.com/mozilla/network/issues/786)
- **Images:** update to 16:9 aspect ratio (low priority) 

I will open these tickets and include checklist of changes, Sketch file, hover states...

**Comment by xmatthewx** [comment]
@kristinashu - this is mostly done, right? if there are important remainders, we should probably set up a new project board with Alan.
